README.md

```diff
@@ -19,9 +19,9 @@ The current list will be maintained by the Radiant Earth team ([@radiantearth](h
 ## Detailed Process
 
 1. Proposals are written up and submitted on <https://github.com/radiantearth/stac-psc/issues/new/choose> for discussion and voting, by any interested party, not just committee members.
-2. Proposals need to be available for review for at least two business days before a final decision can be made. Public holidays count as business days.
+2. Proposals need to be available for review for at least five business days before a final decision can be made. Public holidays count as business days.
 3. Respondents may vote `+1` to indicate support for the proposal and a willingness to support implementation.
-4. Respondents may vote `-1` to veto a proposal, but must provide clear reasoning and alternate approaches to resolving the problem within the two days.
+4. Respondents may vote `-1` to veto a proposal, but must provide clear reasoning and alternate approaches to resolving the problem within the five days.
 5. A vote of `-0` indicates mild disagreement, but has no effect. A `0` indicates no opinion. A `+0` indicates mild support, but has no effect.
 6. Anyone may comment on proposals on the list, but only members of the Project Steering Committee’s votes will be counted.
 7. A proposal will be accepted if it receives +3 (including the proposer) and no vetoes (-1) or has not received enough votes in 20 business days.
```

Comment:
This was recently under discussion in the openEO PSC because it's not 100% clear what this implies:

1. Does a single-1 mean a vote will be rejected after 20 business days? (I think no due to "The absolute majority rule can be used to override an obstructionist veto")
2. Do we always need to wait 20 business days until a decision can be made unless all PSC members have been voted? (I think no due to "Proposals need to be available for review for at least five business days before a final decision can be made. ")

Just want to raise this for awareness. Number 7 is relatively ambiguous.